react/src/components/LegacyFolderExplorer.tsx

```diff
@@ -253,85 +189,120 @@ const LegacyFolderExplorer: React.FC<LegacyFolderExplorerProps> = ({
       }}
       styles={{
         body: {
-          minHeight: '475px',
+          minHeight: token.screenXSMax,
+          // Set height to avoid the modal from growing too large when description is collapsed
+          height: sizes[sizes.length - 1] === 0 ? token.screenXSMax : undefined,
         },
       }}
       {...modalProps}
     >
-      {vfolder_node?.unmanaged_path ? (
-        <Alert
-          message={t('explorer.NoExplorerSupportForUnmanagedFolder')}
-          showIcon
-        />
-      ) : (
-        <>
-          <Flex
-            justify="end"
-            gap={token.marginSM}
-            style={{
-              position: 'absolute',
-              top: `calc(var(--general-modal-header-height, 69px) + 9px)`,
-              right: token.paddingLG,
-            }}
+      <Splitter
```

Comment:
It's not strong suggestion also. It might be good we can use more symantic named component instead of Splitter. It leads us to more readable & understandable code and Many of variabled related to Splitter can be isolated from the the main component. For example:
```
<FolderExplorerSplitterView>
    // Use more symantic named component (I'm not sure about this name) eg. FolderExplorerHeader
    <FolderExplorerHeaderPane alert{vfolder_node?.unmanaged_path}>
         <Flex></Flex>
    </FolderExplorerHeader>
    <FolderNodeDescriptionPane>
          <VFolderNodeDescription />. 
    </FolderNodeDescriptionPane>
</FolderExplorerSplitterView>
```